### PR TITLE
words.txt: Add 400 words for mapping plus codes

### LINKED ISSUE #3: 400 words for mapping plus codes
Get a list of 400 words in a text file, with 1 word per new line to be used for mapping 2 letter segments of a plus code into words.

### PR DESCRIPTION
**Pull request by kx-chen**
Closes https://github.com/MLH-Fellowship/0.5.2-hackathon-geography/issues/3

Please take a quick look and see if there are any words we don't want. I took a quick pass and it seemed all good.

### CODE REVIEW

File: words.txt [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+drive
+stood
+contain
+front
+teach
+week
+final
+gave
+green
+oh
```
Comment (YKo20010) [inline words.txt:336]: ^

File: words.txt [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+little
+only
+round
+man
+year
+came
+show
+every
+good
+me
```
Comment (YKo20010) [inline words.txt:23]: Can we set the min length of words to be 3 char?

File: words.txt [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+change
+went
+light
+kind
+off
+need
+house
+picture
+try
+us
```
Comment (YKo20010) [inline words.txt:91]: ^

File: words.txt [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+might
+story
+saw
+far
+sea
+draw
+left
+late
+run
+don't
```
Comment (YKo20010) [inline words.txt:143]: Prefer not to have '

File: words.txt (hunk 2 above)
Comment (YKo20010) [reply words.txt:23]: On another note, should we have a max length too?

Comment (kx-chen) [reply words.txt:23]: Sure, I think a min and max length is good. min 3, max 8?? suggestions?

Comment (chrisnguyn) [reply words.txt:23]: words <= 8 sound fine to me 👍

**Pull request merged**